rust/Dockerfile

```diff
@@ -11,9 +11,9 @@ FROM rust:${RUST_VERSION}-alpine${ALPINE_VERSION} AS chef
 
 ARG CARGO_CHEF_VERSION
 RUN set -xe \
-  && apk add --no-cache musl-dev  \
-  && cargo install cargo-chef --locked --version=${CARGO_CHEF_VERSION} \
-  && rm -rf $CARGO_HOME/registry/
+    && apk add --no-cache musl-dev  \
+    && cargo install cargo-chef --locked --version=${CARGO_CHEF_VERSION} \
+    && rm -rf $CARGO_HOME/registry/
```

Comment:
This became 4 spaced?

Reply:
Oh, must have been a formatter I have in my editor. Should I revert?